quantecon/random/utilities.py

```diff
@@ -161,3 +161,46 @@ def sample_without_replacement(n, k, num_trials=None, random_state=None):
         return result[0]
     else:
         return result
+
+
+@generated_jit(nopython=True)
+def draw(cdf, size=None):
+    """
+    Generate a random sample according to the cumulative distribution
+    given by `cdf`. Jit-complied by Numba in nonptython mode.
```

Comment:
Did you mean "Jit-compiled by Numba in nopython mode"?

Reply:
Ah good catch, thanks.